internal/core/window.rs

```diff
@@ -972,14 +992,15 @@ impl WindowInner {
             .map_or(ColorScheme::Unknown, |x| x.color_scheme())
     }
 
-    /// Show a popup at the given position relative to the item
+    /// Show a popup at the given position relative to the item and returns its ID.
+    /// The returned ID will always be non-zero.
```

Comment:
that's a candidate for `NonNull<u32>` then.

Reply:
I believe it should be `NonZero<u32>`, but it's a part of the standard library.

Reply:
Right, NonZero. Actually, I meant `core::num::NonZeroU32` which is in core

Reply:
Done